src/posthog-core.js

```diff
@@ -85,6 +85,7 @@ const defaultConfig = () => ({
     // Used for internal testing
     _onCapture: () => {},
     _capture_metrics: false,
+    capture_canvas: false,
```

Comment:
I'm wondering about the naming here: posthog.js does a lot of things so users might think this means we autocapture events on canvas after this.

How about naming this `session_recording_capture_canvas`?